src/main/java/com/kodilla/ecommercee/domains/User.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.ecommercee.domains;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.web.bind.annotation.CrossOrigin;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "USERS")
+public class User {
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "USER_ID")
+    private Long id;
+
+    @Getter
+    @Column(name = "USERNAME")
+    private String username;
+
+    @Getter
+    @Column(name = "STATUS")
+    private String status;
+
+    @Getter
+    @Column(name = "USER_KEY")
+    private Long userKey;
+
+    @Getter
+    @OneToMany(
+            targetEntity = Order.class,
+            mappedBy = "user",
+            cascade = CascadeType.ALL,
+            fetch = FetchType.EAGER
+    )
+    private List<Order> orders = new ArrayList<>();
+}
```

Comment:
Brakuje adnotacji takich jak (cascadeType i fetch) między encjami CART - USER, to chyba najlepiej dodać przy adnotacji OneToOne w encji CART

Reply:
zrobione